### ISSUE TITLE
isFocused() returns `false` immediately after the focus event

### ISSUE THREAD

**Issue by gch1p** [electron/electron#9530]
* Electron version: 1.7.0, 1.6.7
* Operating system: macOS

Run this code in the main process. `win` is the instance of `BrowserWindow`.
```
win.on('focus', () => {
  console.log('isFocused:', win.isFocused())
})
```

### Expected behavior

`isFocused()` should return `true`.

### Actual behavior

It returns `false`.

**Comment by Camyll** [comment]
I'm having the same issue. I'm on Windows Creator's update and electron 1.3.6

**Comment by MarshallOfSound** [comment]
We are no longer implementing bugfixes for versions of Electron <= `1.7.x`, so I'm going to close this issue but if it is still persisting in more recent versions of Electron we can certainly reopen it!

_Side note:_ Pretty sure I've seen this get fixed in `2.0` and `3.0`
